pantheon-tweaks/pantheon-tweaks
# Bump required granite version

Should fix #61

Steps to check if this fixes the linked issue:

1. Download and try to install the latest package of Pantheon Tweaks from http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu/pool/main/p/pantheon-tweaks. You will be asked to install `libgranite5` which does not exist on Odin:

```bash
ryonakano@ryonakano-pc:~$ sudo apt install ./Downloads/pantheon-tweaks_1.0-0_202012170156_ubuntu6.1_amd64.deb 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'pantheon-tweaks' instead of './Downloads/pantheon-tweaks_1.0-0_202012170156_ubuntu6.1_amd64.deb'
The following additional packages will be installed:
  libgranite5
The following NEW packages will be installed:
  libgranite5 pantheon-tweaks
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/248 kB of archives.
After this operation, 1609 kB of additional disk space will be used.
Do you want to continue? [Y/n] 
```

2. Download and try to install the package of Pantheon Tweaks based on this branch from http://ppa.launchpad.net/ryonakaknock3/testing/ubuntu/pool/main/p/pantheon-tweaks. The installation should succeed:

```bash
ryonakano@ryonakano-pc:~$ sudo apt install ./Downloads/pantheon-tweaks_1.0-0_202103040039_ubuntu6.1_amd64.deb 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'pantheon-tweaks' instead of './Downloads/pantheon-tweaks_1.0-0_202103040039_ubuntu6.1_amd64.deb'
The following NEW packages will be installed:
  pantheon-tweaks
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/80.9 kB of archives.
After this operation, 958 kB of additional disk space will be used.
Get:1 /home/ryonakano/Downloads/pantheon-tweaks_1.0-0_202103040039_ubuntu6.1_amd64.deb pantheon-tweaks amd64 1.0-0~202103040039~ubuntu6.1 [80.9 kB]
Selecting previously unselected package pantheon-tweaks.
(Reading database ... 151154 files and directories currently installed.)
Preparing to unpack .../pantheon-tweaks_1.0-0_202103040039_ubuntu6.1_amd64.deb ...
Unpacking pantheon-tweaks (1.0-0~202103040039~ubuntu6.1) ...
Setting up pantheon-tweaks (1.0-0~202103040039~ubuntu6.1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
ryonakano@ryonakano-pc:~$ 
```
